### ISSUE TITLE
Game crashing on next turn for 284th turn

### ISSUE THREAD

**Issue by Sazu4** [yairm210/Unciv#7599]
Tried on android and desktop. Tried to change settings, but no change.

**Platform:** Desktop
**Version:** 4.2.1 (Build 744)
**Rulesets:** [Higher quality builtin sounds, Unciv Vanilla Music Pack, Civ V - Vanilla, Additional Music Ambient, RekMOD, AbsoluteUnits, Medieval Civilizations, DeCiv Redux, 3rd and 4th Unique Component, Civ V Leader portraits, Civ6 mod, Thin Bubbly Borders, Higher Quality City Ambience Sounds, Ancient Civilizations, Civ V - Gods & Kings, 5Hex Tileset, Additional Music Various, Greece plus The Middle East]
**Last Screen:** `com.unciv.ui.worldscreen.WorldScreen`

--------------------------------

OS: Windows 10
Java: Oracle Corporation 25.232-b09
	Max Memory: 910 MB

--------------------------------

**Message:**
```
java.lang.NullPointerException
	at com.unciv.logic.automation.civilization.ReligionAutomation.buyInquisitorNear(ReligionAutomation.kt:170)
	at com.unciv.logic.automation.civilization.ReligionAutomation.spendFaithOnReligion(ReligionAutomation.kt:50)
	at com.unciv.logic.automation.civilization.NextTurnAutomation.automateCivMoves(NextTurnAutomation.kt:66)
	at com.unciv.logic.GameInfo.nextTurn(GameInfo.kt:273)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invokeSuspend(WorldScreen.kt:579)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke(WorldScreen.kt)
	at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke(WorldScreen.kt)
	at com.unciv.utils.concurrency.ConcurrencyKt$launchCrashHandling$1.invokeSuspend(Concurrency.kt:83)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:164)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher$dispatch$1.invoke(Concurrency.kt:164)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandling$1.invoke(CrashHandlingExtensions.kt:17)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.ui.crashhandling.CrashHandlingExtensionsKt$wrapCrashHandlingUnit$1.invoke(CrashHandlingExtensions.kt:33)
	at com.unciv.utils.concurrency.CrashHandlingDispatcher.dispatch$lambda-0(Concurrency.kt:164)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```
**Save Mods:**
```
[Higher quality builtin sounds, RekMOD, Medieval Civilizations, AbsoluteUnits, Thin Bubbly Borders, Civ V Leader portraits, 5Hex Tileset, Greece plus The Middle East]
```
[Mysore 283 Turns Crash issue.txt](https://github.com/yairm210/Unciv/files/9259014/Mysore.283.Turns.Crash.issue.txt)

**Comment by yairm210** [comment]
Looks like what https://github.com/yairm210/Unciv/pull/7581 fixes

**Comment by yairm210** [comment]
Resolved by #7581
